integ-test/build.gradle

```diff
@@ -135,8 +136,11 @@ stopPrometheus.mustRunAfter startPrometheus
 // Run PPL ITs and new, legacy and comparison SQL ITs with new SQL engine enabled
 integTest {
     dependsOn ':opensearch-sql-plugin:bundlePlugin'
-    dependsOn startPrometheus
-    finalizedBy stopPrometheus
+
+    if (DefaultNativePlatform.currentOperatingSystem.isLinux()) {
```

Comment:
Same here.  Maybe a comment why we don't run this for non-Linux.  
Someone might ask why don't we support promethius for all OSes

Reply:
Fixed in 82942faa8.